site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,548 @@
+You can easily embed software comments from multiple languages,
+including Python, Java, and .Net.
+
+Here is an example of embedding Python documentation for a class in your project::
+
+    Make a request
+    --------------
+
+    You can make requests using the :func:`api.request` module.
+    It makes HTTP requests against our website.
+    Here is the basic function signature:
+
+    .. autofunction:: api.request
+
+As you can see,
+you can include prose content along with generated content in the same file.
+This allows for building a narrative around generated code comments,
+instead of giving your users a separate User Guide and API Reference,
+which is often times just a alphabetical listing of code!
+
+Documentation is best written by humans.
+Pulling comments from source code is valuable,
+but it should be a small part of a proper set of documentation.
+
+You can read more about including code comments in the following documentation pages:
+
+* Domains, where the references are defined - http://www.sphinx-doc.org/en/stable/domains.html#what-is-a-domain
+* Autodoc, which generates docs from Python code - http://www.sphinx-doc.org/en/stable/ext/autodoc.html
+* Breathe, which bridges Doxygen and Sphinx - https://breathe.readthedocs.io/en/latest/
+* AutoAPI, which bridges .Net and Sphinx - https://sphinx-autoapi.readthedocs.io/en/latest/
+
+Tables
+------
+
+Working with tables can be the bane of anyone who uses plaintext markup languages.
+Most other languages require that you write them in the file with some arcane syntax.
+However with reStructuredText,
+you can use directives to make this much easier.
+
+It comes with two powerful list directives,
+``csv-table`` and ``list-table``.
+Here is an example of ``csv-table``::
+
+    .. csv-table:: Frozen Delights!
+       :header: "Treat", "Quantity", "Description"
+       :widths: 15, 10, 30
+
+       "Albatross", 2.99, "On a stick!"
+       "Popcorn", 1.99, "Straight from the oven"
+
+This shows the inline markup,
+however the CSV can also be managed in an external file.
+This allows you to manage your complex tables in a third party tool,
+and have your documentation consume them from a CSV which is a much nicer workflow.
+
+Internationalization
+--------------------
+
+Sphinx has support for translating documentation into multiple languages.
+Since sphinx knows the structure of your documents,
+it is able to generate a catalog of content split by each paragraph.
+
+Sphinx internationalization works using the *gettext* system.
+It ships with a builder that allows you to generate a catalog::
+
+    make gettext
+
+You can then use that catalog as the base translation,
+using any tool that supports gettext.
+I recommend using Transifex,
+which gives you a web-based system for translating the documentation:
+
+.. image:: transifex.png
+
+You can then tell Sphinx what language to generate for it's documentation when you build it by setting the ``language`` setting.
```

Comment:
"its" (apostrophe misplaced, and yes everyone does it all the time but this way you can look Especially Literate :D )